src/Backend/Modules/MediaLibrary/Config.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Backend\Modules\MediaLibrary;
+
+use Backend\Core\Engine\Base\Config as BackendBaseConfig;
+
+/**
+ * This is the configuration-object for the media module
+ */
+class Config extends BackendBaseConfig
```

Comment:
the things you defined are the defaults, you can leave both `$defaultAction` and `$disabledActions` away